docs/apache-airflow/howto/operator/bash.rst

```diff
@@ -79,7 +79,7 @@ Skipping
 
 In general a non-zero exit code produces an AirflowException and thus a task failure.  In cases where it is desirable
 to instead have the task end in a ``skipped`` state, you can exit with code ``99`` (or with another exit code if you
-pass ``skip_exit_code``).
+pass ``skip_on_exit_code``).
```

Comment:
Better to move it into the separate PR, because BashOperator is part of Airflow core and it has a different release cycle. It will make release managers happier 😉

Reply:
Ok